presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries
+  that enable spilling and that are moved to the reserved memory pool.
+* Fix a bug that can cause the planner to go into an infinite loop. This bug was affecting queries
```

Comment:
Move the bug fixes to the top. I generally put bug fixes, performance improvements and new features in that order.

Reply:
Fix an infinite loop during planning for queries containing non-trivial predicates.